docs/spec-files/appendix.md

```diff
@@ -0,0 +1,671 @@
+## Appendix
+This section provides test vectors of DIDComm Messages. The test vectors defined in the [Appendix C](#appendix-c-test-vectors) MIGHT be used to validate implementations of [DIDComm Messaging](.). The test vectors  MUST use the sender and recipient private keys defined in the [Appendix A](#appendix-a-secrets-for-test-vectors) to encrypt and sign DIDComm Messages. The test vectors MUST use the sender and recipient public keys defined in the [Appendix B](#appendix-b-diddocs-for-test-vectors) to decrypt DIDComm Encrypted Messages and verify signature of DIDComm Signed Messages.
```

Comment:
> The test vectors  MUST use the sender and recipient **private keys** defined in the [Appendix A](#appendix-a-secrets-for-test-vectors) to **encrypt** and sign DIDComm Messages 

NB: a private key is used to decrypt, not to encrypt.

> The test vectors MUST use the sender and recipient **public keys** defined in the [Appendix B](#appendix-b-diddocs-for-test-vectors) to **decrypt** DIDComm Encrypted Messages and verify signature of DIDComm Signed Messages.

And a public key is used to encrypt, not to decrypt, when talking about encryption.